src/app/pages/ArticlePage/PagePromoSections/TopStoriesSection/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { arrayOf, shape } from 'prop-types';
+import { storyItem } from '#models/propTypes/storyItem';
+import pathOr from 'ramda/src/pathOr';
+import { C_GREY_2 } from '#app/legacy/psammead-styles/src/colours';
+import SectionLabel from '#legacy/psammead-section-label/src';
+import isEmpty from 'ramda/src/isEmpty';
+import {
+  StyledWrapper,
+  FlexPromoList,
+  FlexPromoListItem,
+} from './index.styles';
+import TopStoriesItem from './TopStoriesItem';
+import optimoPromoIdGenerator from '../utility';
+
+const TopStoriesSection = ({ content }) => {
+  const { translations, script, service } = useContext(ServiceContext);
+
+  if (!content || isEmpty(content)) return null;
+
+  const title = pathOr('Top Stories', ['topStoriesTitle'], translations);
+  const hasSingleContent = content.length === 1;
+  const LABEL_ID = 'top-stories-heading';
+
+  const singleItem = () => {
```

Comment:
`singleItem` for a function is not a great name. `renderSingleItem` would be better. However, I don't think should be a function at all

Reply:
This might be another of my 🤦‍♂️ moments. Will try to remove the function. I was trying to have the function be inline at first but did not work out.

Reply:
removed function and added function for mapping items